src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,4 @@
+@mixin box-size {
+  height: $box-height;
+  width: $box-width;
+}
```

Comment:
you can use mixin with a parameter, so you don't need to create extra varible:
@mixin box ($size) {
  width: $size;
  height: $size;
}

and instead of 
@include box-size;
you can write 
@include box($box-width);